packages/manager/modules/pci/src/projects/project/notebooks/add/add.html

```diff
@@ -1,13 +1,136 @@
-<oui-header data-heading="{{:: 'pci_notebook_add_title' | translate }}">
-    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
-        <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
-            <span data-translate="pci_project_guides_header_all_guides"></span>
-        </oui-guide-menu-item>
-    </oui-guide-menu>
-</oui-header>
+<div>
+    <oui-header data-heading="{{:: 'pci_notebook_add_title' | translate }}">
+        <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
+            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+                <span data-translate="pci_project_guides_header_all_guides"></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-header>
 
+    <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent aliquet tellus enim, vel porttitor nulla pharetra vel.
+        Praesent iaculis enim eu lacus dapibus bibendum. Cras in ex aliquam, eleifend arcu et hendrerit quam. Quisque fermentum bibendum lectus.
+        Cras purus dolor, fermentum sit amet vulputate id, pretium quis lorem.
+    </p>
+</div>
+
+<!--Banner messages-->
 <div class="mt-4" id="addMessages">
+    <oui-message data-type="info">
+        <span data-translate="pci_notebook_add_banner_object_storage_description"></span>
+        <a
+            data-ui-sref="pci.projects.project.storages"
```

Comment:
Use `ng-href`, with the link generated in the resolver instead. We stopped using the state names in the controller/html templates